unblob/cli.py

```diff
@@ -54,5 +55,21 @@ def main():
         cli.invoke(ctx)
 
 
+def main():
```

Comment:
I don't see the point of having two "main" functions. Just wrap the `cli.invoke` with try-except above. It would be much more linear.

Reply:
I made the changes, but I wrapped everything, not just the `cli.invoke`. The whole point is to be able to handle any possible exception raised - even in `cli.make_context`, which is currently only handling specific exceptions.